File: Deletion/delete.js
let arr = [];
let n = parseInt(prompt("Enter the number of elements: "));

console.log("Enter the elements:");
for (let i = 0; i < n; i++) {
    arr.push(parseInt(prompt()));
}

let index_pos = parseInt(prompt("Enter the index where the element is to be deleted: "));

// Shift elements to the left to make space for the deleted element
for (let i = index_pos; i < n - 1; i++) {
    arr[i] = arr[i + 1];
}

console.log("The elements entered are: " + arr.slice(0, n - 1).join(" "));

File: Deletion/delete.c

#include <stdio.h>

#define max 100

int main() {
    int arr[max];
    int n, i;
    int index = -1;
    int element;

    printf("Enter the number of elements (up to %d): ", max);
    scanf("%d", &n);

    printf("Enter the elements:\n");
    for (i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    
    while(index == -1){

        printf("Enter the element is to be deleted: ");
        scanf("%d", &element);

        for(i = 0; i<n ; i++){
            if(arr[i] == element){
                index = i;
                break;
            }
        }
        printf("The element is not in the array\n");
        index = 1;
    }


    // Shift elements to the left to destroy space for the deleted element
    if(index != -1 || index != 1){
        for (i = index; i < n-1; i++) {
            arr[i] = arr[i+1];
        }
    }

    printf("The elements entered are: ");
    for (i = 0; i < n-1; i++) {
        printf("%d ", arr[i]);
    }
    printf("\n");

    return 0;
}
